tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -64,7 +69,7 @@ def ignore_loganalyzer_exceptions(enum_rand_one_per_hwsku_frontend_hostname, log
 
 @pytest.fixture(scope='class', autouse=True)
 def pfcwd_timer_setup_restore(setup_pfc_test, enum_fanout_graph_facts, duthosts,        # noqa F811
-                              enum_rand_one_per_hwsku_frontend_hostname, fanouthosts):
+                              enum_rand_one_per_hwsku_frontend_hostname, fanouthosts, stop_pfcwd):
```

Comment:
For this fixture here, I remove the autouse since the behavior is unexected. We want our fixture to be this following order:

1. stop_pfcwd
2. pfcwd_timer_setup_restore
3. pfcwd_timer_setup_restore clean up
4. stop_pfcwd clean up

This change here will make sure that this happen.